.appveyor.yml

```diff
@@ -20,5 +20,5 @@ build: off
 
 test_script:
   - C:\\msys64\\usr\\bin\\bash --login -c "python -m unittest discover -s %WEBOTS_HOME_PROTECTED%/docs/tests"
-  - ps: if ($env:APPVEYOR_SCHEDULED_BUILD -ne "True" -and $env:APPVEYOR_PULL_REQUEST_NUMBER -ge 1) { C:\\msys64\\usr\\bin\\bash --login -c "python $env:WEBOTS_HOME_PROTECTED/tests/sources/generate_diff.py $env:APPVEYOR_REPO_BRANCH" }
+  - ps: if ($env:APPVEYOR_SCHEDULED_BUILD -ne "True" -and $env:APPVEYOR_REPO_TAG -ne "True") { C:\\Python37-x64\\python.exe $env:WEBOTS_HOME\\tests\\sources\\generate_diff.py $env:APPVEYOR_REPO_COMMIT $env:APPVEYOR_REPO_NAME }
```

Comment:
Are you sure about the capitalization of `True` for `APPVEYOR_REPO_TAG`? From the doc it seems it is lowercase:

> APPVEYOR_REPO_TAG - true if build has started by pushed tag; otherwise false
https://www.appveyor.com/docs/environment-variables/